src/syscheckd/db/src/fimDB.cpp

```diff
@@ -34,47 +32,26 @@ void FIMDB::setValueLimit()
 
 void FIMDB::registerRSync()
 {
-    // LCOV_EXCL_START
-    const auto reportFimSyncWrapper
-    {
-        [this](const std::string & dataString)
-        {
-            m_syncMessageFunction(dataString);
-            m_loggingFunction(LOG_DEBUG_VERBOSE, "Sync sent: " + dataString);
-        }
-    };
-    // LCOV_EXCL_STOP
-
-    m_rsyncHandler->registerSyncID("fim_file_sync",
+    m_rsyncHandler->registerSyncID("fim_file",
                                     m_dbsyncHandler->handle(),
                                     nlohmann::json::parse(FIM_FILE_SYNC_CONFIG_STATEMENT),
-                                    reportFimSyncWrapper);
+                                    m_syncFileMessageFunction);
 #ifdef WIN32
-    m_rsyncHandler->registerSyncID("fim_registry_sync",
+    m_rsyncHandler->registerSyncID("fim_registry",
```

Comment:
You must use a constant to "fim_registry"

Reply:
done